sql/slow-query.md

```diff
@@ -8,87 +8,209 @@ category: user guide
+If you want to query the same SQL fingerprint query after querying TopN's SQL, you can use the fingerprint as the filter condition.
 
-#### `table_ids`
+```sql
+tidb > select query_time, query,digest from INFORMATION_SCHEMA.`SLOW_QUERY` where `Is_internal`=false order by `Query_time` desc limit 1;
++-------------+-----------------------------+------------------------------------------------------------------+
+| query_time  | query                       | digest                                                           |
++-------------+-----------------------------+------------------------------------------------------------------+
+| 0.302558006 | select * from t1 where a=1; | 4751cb6008fda383e22dacb601fde85425dc8f8cf669338d55d944bafb46a6fa |
++-------------+-----------------------------+------------------------------------------------------------------+
+1 row in set
+Time: 0.007s
+tidb > select query, query_time from INFORMATION_SCHEMA.`SLOW_QUERY` where digest="4751cb6008fda383e22dacb601fde85425dc8f8cf669338d55d944bafb46a6fa";
++-----------------------------+-------------+
+| query                       | query_time  |
++-----------------------------+-------------+
+| select * from t1 where a=1; | 0.302558006 |
+| select * from t1 where a=2; | 0.401313532 |
++-----------------------------+-------------+
+2 rows in set
+```
 
-The IDs of the tables involved in the statement.
+## Query the slow query with pseudo metrics 
 
-#### `index_ids`
+```sql
+tidb > select query, query_time, stats from INFORMATION_SCHEMA.`SLOW_QUERY` where is_internal=false and stats like('%pseudo%');
++-----------------------------+-------------+---------------------------------+
+| query                       | query_time  | stats                           |
++-----------------------------+-------------+---------------------------------+
+| select * from t1 where a=1; | 0.302558006 | t1:pseudo                       |
+| select * from t1 where a=2; | 0.401313532 | t1:pseudo                       |
+| select * from t1 where a>2; | 0.602011247 | t1:pseudo                       |
+| select * from t1 where a>3; | 0.50077719  | t1:pseudo                       |
+| select * from t1 join t2;   | 0.931260518 | t1:407872303825682445,t2:pseudo |
++-----------------------------+-------------+---------------------------------+
+```
 
-The IDs of the indexes involved in the statement.
+#### Parse other TiDB slow log files
 
-#### `sql`
+Currently, query `INFORMATION_SCHEMA.SLOW_QUERY` will only parse the slow log file name set by `slow-query-file` in the configuration file, which is set to "tidb-slow.log" by default. But if you want to parse other log files, you can set the session variable `tidb_slow_query_file` to the specific file path, and then query INFORMATION_SCHEMA.SLOW_QUERY` to parse the slow log file according to the set path.
+```sql
+/* Set the slow log file path to facilitate parsing other slow log files. The scope of the tidb_slow_query_file variable is session. */
```

Comment:
```suggestion
/* Set the slow log file path to facilitate so that other slow log files will be easy to be parsed. The scope of the tidb_slow_query_file variable is session. */
```